bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tf1.py

```diff
@@ -0,0 +1,140 @@
+# Author: Aaron Jomy CERN 08/2024
+
+################################################################################
+# Copyright (C) 1995-2022, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+r"""
+/**
+\class TF1
+\brief \parblock \endparblock
+\htmlonly
+<div class="pyrootbox">
+\endhtmlonly
+## PyROOT
+
+The TF1 class has several additions for its use from Python, which are also
+available in its subclasses TF2 and TF3.
+
+First, TF1 instance can be inititialised with user-defined Python functions. Given a functor object or a pycallable,
```

Comment:
```suggestion
First, TF1 instance can be initialised with user-defined Python functions. Given a generic Python callable,
```

I believe the notion of "functor" might be too specific for a general user, in any case it's a callable.